linux_os/guide/system/accounts/accounts-pam/set_password_hashing_algorithm/set_password_hashing_algorithm_systemauth/oval/shared.xml

```diff
@@ -13,17 +13,49 @@
     </metadata>
     <criteria operator="AND">
       <criterion test_ref="test_pam_unix_sha512" />
+	  <criterion test_ref="test_pam_unix_noncompliant_hash" negate="true" />
+	  <criterion test_ref="test_password_auth_pam_unix_sha512" />
+	  <criterion test_ref="test_password_auth_pam_unix_noncompliant_hash" negate="true" />
```

Comment:
Please make sure that this format is whitespace and not tabs and remove the leading two spaces.